sdk/endpoint/compositeBuilder.go

```diff
@@ -0,0 +1,24 @@
+package endpoint
```

Comment:
Please add licensing here.

Reply:
And well add these in all yuor new files.

Reply:
@nickolaev I wonder if we could lint for license headers?

Reply:
@edwarnicke actually, we have not the linter for license headers. We can create an issue for this problem and I can solve this in separate PR.

Reply:
+1